EIPS/eip-5792.md

```diff
@@ -43,6 +43,8 @@ These fields are set inside this capability's entry in the `capabilities` object
 Each entity in the `calls` field may contain an optional `capabilities` object.
 This object allows the applications to attach a capability-specific metadata to individual calls.
 
+The `atomicity` field specifies how the wallet should handle the batch of calls. Despite not being a capability, and being a top-level field in the request, the wallet supported values can be sourced via `wallet_getCapabilities`.
```

Comment:
> Despite not being a capability

This is inconsistent with the later section titled "`atomicity` Capability". Should this just say something like the following instead?

```suggestion
The `atomicity` field specifies how the wallet should handle the batch of calls. It is a built in capability and the wallet supported values can be sourced via `wallet_getCapabilities`.
```